From 35d2c2018371099fc07fdf3ddda4d37783705411 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fatih=20A=C5=9F=C4=B1c=C4=B1?= <fatih.asici@gmail.com>
Date: Sat, 5 Oct 2013 22:57:41 +0300
Subject: [PATCH] examples: Fix build when configured with -no-gui
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Building of examples fails when configured with -no-gui. This patch adds gui
dependency to the relevant .pro files.

Reported-Upstream: https://bugreports.qt-project.org/browse/QTBUG-33910
Signed-off-by: Fatih Aşıcı <fatih.asici@gmail.com>
---
 examples/gui/gui.pro                   | 2 ++
 examples/qpa/qpa.pro                   | 2 ++
 examples/qtconcurrent/qtconcurrent.pro | 1 +
 examples/threads/threads.pro           | 2 ++
 4 files changed, 7 insertions(+)

diff --git a/examples/gui/gui.pro b/examples/gui/gui.pro
index 8758a1b..5fb4241 100644
--- a/examples/gui/gui.pro
+++ b/examples/gui/gui.pro
@@ -1,3 +1,5 @@
+requires(qtHaveModule(gui))
+
 TEMPLATE     = subdirs
 CONFIG += no_docs_target
 
diff --git a/examples/qpa/qpa.pro b/examples/qpa/qpa.pro
index 6f07e50..2729348 100644
--- a/examples/qpa/qpa.pro
+++ b/examples/qpa/qpa.pro
@@ -1,2 +1,4 @@
+requires(qtHaveModule(gui))
+
 TEMPLATE      = subdirs
 SUBDIRS       = windows
diff --git a/examples/qtconcurrent/qtconcurrent.pro b/examples/qtconcurrent/qtconcurrent.pro
index 1df6f2d..5e8c77f 100644
--- a/examples/qtconcurrent/qtconcurrent.pro
+++ b/examples/qtconcurrent/qtconcurrent.pro
@@ -1,4 +1,5 @@
 requires(qtHaveModule(concurrent))
+requires(qtHaveModule(gui))
 
 TEMPLATE      = subdirs
 SUBDIRS       = imagescaling \
diff --git a/examples/threads/threads.pro b/examples/threads/threads.pro
index e47da84..b5cf12a 100644
--- a/examples/threads/threads.pro
+++ b/examples/threads/threads.pro
@@ -1,3 +1,5 @@
+requires(qtHaveModule(gui))
+
 TEMPLATE      = subdirs
 CONFIG += no_docs_target
 
-- 
1.8.4.rc3

